templates/forum-ng-submitted.tpl.php

```diff
@@ -19,12 +20,13 @@
  */
 ?>
 <?php if ($time): ?>
-  <span class="submitted">
-  <?php print t('By !author @time ago', array(
-    '@time' => $time,
-    '!author' => $author,
-    )); ?>
-  </span>
+  <div class="submitted forum-ng-submitted">
+    <?php print $author_picture; ?>
```

Comment:
Not sure about the image there without any float or styling. Comment on the issue coming up.

Reply:
I don't think that this is what is causing that display. I'll respond in the comment/issue.